src/pudl/glue/ferc1_eia.py

```diff
@@ -451,20 +461,21 @@ def get_util_ids_eia_unmapped(
     utilities_eia_db = pudl_out.utils_eia860()[
         ["utility_id_eia", "utility_name_eia"]
     ].drop_duplicates(["utility_id_eia"])
-    unmapped_utils_eia = get_missing_ids(
+    unmapped_utils_eia_index = get_missing_ids(
         utilities_eia_mapped, utilities_eia_db, id_cols=["utility_id_eia"]
     )
 
     # Get the most recent total capacity for the unmapped utils.
-    unmapped_utils_eia = (
-        get_utility_most_recent_capacity(pudl_engine)
-        .loc[unmapped_utils_eia]
-        .merge(
-            utilities_eia_db.set_index("utility_id_eia"),
-            left_index=True,
-            right_index=True,
-            how="left",
-        )
+    utilities_eia_db = utilities_eia_db.set_index(["utility_id_eia"])
+    unmapped_utils_eia = utilities_eia_db.loc[unmapped_utils_eia_index]
+    util_recent_cap = get_utility_most_recent_capacity(pudl_engine)
+
+    unmapped_utils_eia = pd.merge(
+        unmapped_utils_eia,
+        util_recent_cap,
+        left_index=True,
+        right_index=True,
+        how="left",
```

Comment:
This change was because there were some new utilities that don't show up in the `generators_eia860` table which is where the capacity is coming from and the original `.loc[unmapped_utils_eia] was failing because there were util values not in the capacity table.